src/algebra/group/prod.lean

```diff
@@ -0,0 +1,232 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon, Patrick Massot, Yury Kudryashov
+-/
+
+import algebra.group.hom
+
+variables {A : Type*} {B : Type*} {G : Type*} {H : Type*} {M : Type*} {N : Type*} {P : Type*}
+
+namespace prod
+
+-- TODO (Lean 3.7.0) : make `to_additive` aware of `protected`, and use it for instances
```

Comment:
I lost track. What is the current status of this feature?

Reply:
I think part of it landed in #2212?

Reply:
Ok, so then we should do the todo (-;